Add authentication configurations support (#49)
ucsd-ets/openedx-caliper-tracking: README.rst

    }

2. Add the following keys and their values in the ``lms.env.json`` and ``cms.env.json`` files.
Please note that all parameters in the `PRODUCER_CONFIG` are unique to the broker instances. You

> can set whatever parameters are required for you.

Reviewer: nit:
```suggestion
can set whatever parameters required for your instance.
```

Author: Updated 'you' with 'your instance' but don't think 'are' should be removed.

::

    "CALIPER_KAFKA_SETTINGS": {
        "PRODUCER_CONFIG": {
            "bootstrap_servers": [
                "<List of Kafka Brokers URLs>"
            ],
            ...
        },

        "TOPIC_NAME": "<Kafka Topic Name>",

>         "ERROR_REPORT_EMAIL": "<Reporting email address>",

Reviewer: For consistency:
```suggestion
        "ERROR_REPORT_EMAIL": "<Reporting Email Address>",
```

Author: Done
